packages/babel-types/scripts/generators/docs.js

```diff
@@ -176,6 +176,7 @@ function generateMapAliasToNodeTypes() {
   return result;
 }
 const aliasDescriptions = {
+  Accessor: "A cover of ClassAccessorProperty.",
```

Comment:
The `Accessor` currently only covers ClassAccessorProperty. I believe it used to cover `ClassAccessorProperty` and `ClassPrivateAccessorProperty` but later we merged these two. We may consider remove it in Babel 8.

Reply:
Yes, we should remove it in Babel 8.